features/sponsorshipBenefit.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Sponsorship Benefit
+  In order to test CRUD on sponsorship benefits
+  As an admin
+  I need to be able to list all benefits and fill forms
+
+  Scenario: I add a sponsorship benefit
+    Given I am logged in as an admin
+    When I am on "sponsorship-benefit/create"
+    And I fill in "sponsorship_benefit[label]" with "Benefit 11"
```

Comment:
Too technical
```suggestion
    And I fill the sponsorship benefit label field with an existing benefit
```
Do not leak the tech details here, it's supposed to be plain English here